src/rime/gear/selector.cc

```diff
@@ -149,7 +149,8 @@ ProcessResult Selector::ProcessKeyEvent(const KeyEvent& key_event) {
   else if (ch >= XK_KP_0 && ch <= XK_KP_9)
     index = ((ch - XK_KP_0) + 9) % 10;
   if (index >= 0) {
-    SelectCandidateAt(ctx, index);
+    key_event.alt() ? HiliteCandidateAt(ctx, index)
```

Comment:
I'm not in favour of this feature. Why would user want to do that with a key combo?

Reply:
> I'm not in favour of this feature. Why would user want to do that with a key combo?

because, when intended to be followed by deleting the candidate, it's faster to use a key combo to jump from highlighting candidate 1 to highlighting candidate 10 than press down/right key 9 times, provided that user does not want to use mouse.